lib/resources/dstu2/soarian_care_plan.yaml

```diff
@@ -0,0 +1,5 @@
+---
+name: CarePlan
+field_name_base_url: http://hl7.org/fhir/DSTU2/careplan-definitions.html#CarePlan
```

Comment:
Its an identifier assigned by Soarian, FHIR cardinality allows the “type” as optional.